src/tr1/game/objects/creatures/natla.c

```diff
@@ -206,9 +206,9 @@ static void M_Control(const int16_t item_num)
             if (!(natla->flags & NATLA_FLY_MODE)) {
                 Creature_Mood(item, &info, true);
             }
-            natla->lot.step = WALL_L * 20;
-            natla->lot.drop = -WALL_L * 20;
-            natla->lot.fly = STEP_L / 8;
+            natla->lot.setup.step = WALL_L * 20;
+            natla->lot.setup.drop = -WALL_L * 20;
+            natla->lot.setup.fly = STEP_L / 8;
```

Comment:
Let's make these maybe `static LOT_SETUP` variables and assign entire `setup`?

Reply:
Thanks, this would be neater but after thinking through it I think we should leave it as is. My thinking is if you look in `mutant.c`, similar happens there, but the problem is flyers have a different block mask to non-flyers but the same control is used to change the lot values. If we were to assign the entire struct during control, we'd have to be careful with the block mask too. I know that's not relevant to Natla and the Xian guards, but perhaps for consistency it's OK to have this as it is?

Or an alternative approach could be splitting out step, drop and fly into a sub struct, which would be safer to replace.

Reply:
I think it's fine to leave it as is :)